Dockerfile

```diff
@@ -16,7 +16,6 @@ RUN apk add --no-cache \
     source $HOME/.cargo/env && \
     pip install --no-cache-dir poetry==${POETRY_VERSION} && \
     apk del \
-        curl \
```

Comment:
@thegentlemanphysicist any reason to remove this? I thought this pkg would help debugging